model/Core/Classes/PackageVerificationCode.md

```diff
@@ -18,19 +18,21 @@ This identifier enables a recipient to determine if any file in the original pac
 
 Algorithm:
 
+    hashValue = 0
     templist = empty list of strings
 
-    for all files in the package
-        if file is an "excludes" file
+    for all files in the package {
+        if file is a packageVerificationCodeExcludedFile
             skip it /* exclude SPDX analysis file(s) */
         else
-            append algorithm(contents of file) to templist
+            append "algorithm(file)/n" to templist
+        }
```

Comment:
Not needed, indentation works.

Reply:
Style preference.   Original had {, };   this should have been changed earlier and been in a separate PR.